plotgui.py

```diff
@@ -147,12 +168,17 @@ def mouseMoveEvent(self, event):
         id, properties, domain, domain_kind = self.getIDinfo(event)
         if self.ax.contains_point((event.pos().x(), event.pos().y())):
 
+            if domain_kind.lower() in _MODEL_PROPERTIES:
+                line_val = float(properties[domain_kind.lower()])
+                line_val = max(line_val, 0.0)
+                self.updateDataIndicatorValue(line_val)
+
             if id == str(_VOID_REGION):
                 domainInfo = ("VOID")
             elif id != str(_NOT_FOUND) and domain[id].name:
                 domainInfo = (f"{domain_kind} {id}: \"{domain[id].name}\"\t "
-                             f"Density: {properties['density']} g/cm3\t"
-                             f"Temperature: {properties['temperature']} K")
+                              f"Density: {properties['density']} g/cm3\t"
+                              f"Temperature: {properties['temperature']} K")
```

Comment:
If we're going to merge with OpenMC proper, we need to support Python 3.4 and 3.5 (which don't have f-strings sadly).

Reply:
Just took care of these.